### PR TITLE
Feature/DTGB-155: Topic teaser overview page.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+        teaser_tag: 'Just some tag'
+      },
+      {
+        teaser_uid: 'uid',
+        teaser_paragraph_text: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam corporis cumque debitis eligendi harum iste laborum quibusdam ut, vitae voluptate?',
+        teaser_title: 'News article title text',
+        teaser_event_date: true,
+        text_1: 'From the 7th',
+        datetime_1: '2019-01-07',
+        text_2: 'untill the 13th of January 2019',
```
Comment (delrueba) [inline styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: until

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
```
Comment (delrueba) [inline styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: until

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
```
Comment (delrueba) [inline styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: until

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
```
Comment (delrueba) [inline styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: until

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js (hunk 1 above)
Comment (meirege) [reply styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: Fixed.

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js (hunk 2 above)
Comment (meirege) [reply styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: Fixed.

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js (hunk 3 above)
Comment (meirege) [reply styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: Fixed.

File: styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js (hunk 4 above)
Comment (meirege) [reply styleguide/components/71-pages/general/news-overview-page/news-overview-page.config.js]: Fixed.

**Review by delrueba — APPROVED** [review]

**Pull request merged**